app/controllers/claims/schools/claims_controller.rb

```diff
@@ -2,7 +2,7 @@ class Claims::Schools::ClaimsController < ApplicationController
   include Claims::BelongsToSchool
 
   def index
-    @claims = @school.claims
+    @pagy, @claims = pagy_array(@school.claims.map(&:decorate))
```

Comment:
Hmm, I think we can do the following instead:

```ruby
@pagy, @claims = pagy(@school.claims)
@claims        = @claims.decorate
```

Based on the [documentation](https://github.com/ddnexus/pagy/blob/master/docs/extras/array.md), we shouldn't use persistent storage backed collections. I believe it will attempt to load in _all_ records behind the scenes.

Reply:
Decorating is not necessary anymore as I can get the status tag from `Claim::StatusTagComponent`